recipe/meta.yaml

```diff
@@ -14,13 +13,17 @@ package:
 
 source:
   url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: ba983dbb91a7f51553315fe14f1fbcbb4920f8f2e9bb47620735d2d63c33e673
+  sha256: ee7e3df43da7d924207e03efa4d5e5466801510b3e1f0377d425c6e6a8d823bb
 
 build:
-  number: 2
+  number: 0
   {% if not (environ.get("cuda_compiler_version")|string()).startswith(major_version|string()) %}
   skip: true
   {% endif %}
+  # `linux_ppc64le` is unsupported by the CUDA Toolkit as of 12.5.0.
+  # So skip building `cuda-python` on `linux_ppc64le`
+  # xref: https://github.com/conda-forge/cuda-feedstock/issues/27
+  skip: true  # [ppc64le]
```

Comment:
Note that `linux_ppc64le` is dropped at the CUDA Toolkit no longer `linux_ppc64le` in 12.5.0

As `cuda-python` versions need to be built with the corresponding CUDA Toolkit, it is not possible to build `cuda-python` version `12.5.0` without CUDA Toolkit 12.5.0

xref: https://github.com/conda-forge/cuda-feedstock/issues/27